examples/backend-server/frontend-src/main.js

```diff
@@ -1,7 +1,12 @@
 import 'vite/modulepreload-polyfill'
 import './style.css'
+import okPng from "./ok.png";
 
 document.querySelector('#app').innerHTML = `
   <h1>Hello Vite!</h1>
   <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
+  <pre>import okPng from "./ok.png"</pre>
+  <img src="${okPng}" alt="NOT OK" width="30" />
+  <pre>new URL('./ok.png', import.meta.url).href</pre>
+  <img src="${new URL('./ok.png', import.meta.url).href}" alt="NOT OK" width="30" />
```

Comment:
```suggestion
  <img src="${viteSvg}" alt="Vite Logo" width="60" />
  <h1>Hello Vite!</h1>
  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
```
I think we don't need the example for `new URL()` one as it's not affected by the setup.

I think using the Vite logo looks more polished.
https://github.com/vitejs/vite/blob/main/packages/create-vite/template-vanilla/public/vite.svg

Would you also replace other `main.js` in the `examples` directory to make it consistent?

Reply:
Thanks, that looks better! I was almost turning this example into test cases :sweat_smile: 
I updated other examples as well.